File: front-end/frontBlog/src/app/entities/Category.ts
export class Category {
  categoryId: Number;
  name: string;
  modifiedDate: Date;

  constructor(categoryId: Number, name: string, modifiedDate: Date) {
    this.categoryId = categoryId;
    this.name = name;
    this.modifiedDate = modifiedDate;
  }
}

File: back-end/ponton/src/main/java/com/example/demo/entities/Category.java
package com.example.demo.entities;

import lombok.AccessLevel;
import lombok.Data;
import lombok.Setter;
import org.hibernate.annotations.UpdateTimestamp;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import java.io.Serializable;
import java.util.Date;

@Entity
@Data
public class Category implements Serializable {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Setter(AccessLevel.NONE)
    private Integer categoryId;

    private String name;

    @UpdateTimestamp
    @Setter(AccessLevel.NONE)
    private Date modifiedDate;
}
